### PR TITLE
Module substript to install NLP and CV requirements separately.

### PR DESCRIPTION
**Pull request by venkata-ramireddy-mettu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,24 @@ State-of-the-art models in AI,ML,NLP & Vision for FinTech community by Yubi Data
...
+* Install only Vision dependencies
+```
+pip install ".[cv]"
+```
+* Install both NLP & Vision dependencies
+```
+pip install ".[nlp,cv]"
+```
+
 ## How to use package
```
Comment (swapaj) [inline README.md:28]: You need to change this as well. With and without installing pkg codes. The following codes won't work.

Comment (swapaj) [reply README.md:28]: You also need to change all readme pkg links in `Whats New in NLP and CV` part .. they also won't work.

Comment (swapaj) [reply README.md:28]: Check .. all possible locations where links or classes are called .. all of them have to be corrected

Comment (venkata-ramireddy-mettu) [reply README.md:28]: Done.

**Review by swapaj — APPROVED** [review]

**Pull request merged**